docs/services/navigation/_index.md

```diff
@@ -0,0 +1,268 @@
+---
+title: "The Navigation Service"
+linkTitle: "Navigation"
+description: "The Navigation Service uses GPS to autonomously navigate a rover to user defined endpoints."
+type: docs
+weight: 45
```

Comment:
let's put that after the ML Model service - we should probably alphabetize them. We'll also need to fix the motion one but can do that separately

Reply:
oh great point, done!